lectures/oop_intro.md

```diff
@@ -298,5 +820,90 @@ import math
 id(math)
 ```
 
-This uniform treatment of data in Python (everything is an object) helps keep the language simple and consistent.
+We can find `math` in our global namespace after the import
+
+```{code-cell} python3
+print(dir()[-1::-1])
+```
+
+We can also find all objects associated with the `math` module in the private namespace of `math`
+
+```{code-cell} python3
+print(dir(math))
+```
+
+We can also directly import objects to our current namespace using `from ... import ...`
+
+```{code-cell} python3
+from math import log, pi, sqrt
+
+print(dir()[-1::-1])
+```
+
+We can find these names appear in the current namespace now.
+
+*This uniform treatment of data in Python (everything is an object) helps keep the language simple and consistent.*
+
+## Exercises
+
+```{exercise-start}
+:label: oop_intro_ex1
+```
+
+We have met the {any}`boolean data type <boolean>` previously. 
+Using what we have learnt in this lecture, print a list of methods of boolean objects.
+
+(hint: you can use `callable()` to test whether an attribute of an object can be called as a function)
```

Comment:
@AakashGfude can we use `hint` admonitions in `quantecon-book-theme`?

Reply:
Thanks for asking for me @mmcky, I found we could use hint admonitions. I set it as a dropdown object so that it looks nicer and it gives readers the choice not to look at them.